docs/marketplace/dev/feature-integration-guides/202108.0/merchant-portal-marketplace-product-feature-integration.md

```diff
@@ -36,10 +36,52 @@ Make sure that the following modules have been installed:
 |-|-|
 | ProductMerchantPortalGui | spryker/product-merchant-portal-gui |
 | ProductMerchantPortalGuiExtension | spryker/product-merchant-portal-gui-extension |
+| AvailabilityMerchantPortalGui | spryker/availability-merchant-portal-gui |
```

Comment:
Why? This module doesn't belong to this feature. We should have an additional IG - `Merchant Portal - Marketplace Product + Inventory Management feature integration` where spryker/availability-merchant-portal-gui will be added and the plugin registered. 
See `Merchant Portal - Marketplace Product + Tax feature integration` as example.
See https://spryker.atlassian.net/wiki/spaces/DOCS/pages/478118714/Feature+integration+guides - `Optional feature` section where is described how to build such IGs